### ISSUE TITLE
`array_key_exists` costs less than `isset`

### ISSUE THREAD

**Issue by saulfautley** [kalessil/phpinspectionsea#1403]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended) 3.0.15 |
| Language level | All                        |

Current behaviour
--
```php
if (isset($array[0]) || array_key_exists(0, $array)) {...}
```
Reports "This condition costs less than the previous one." on `array_key_exists`.

Expected behaviour
--
I've done pretty extensive research and testing on this. Best case `array_key_exists` takes 2x as long as as `isset`, and even longer on older PHP versions.

It looks like this was fixed in #670 in 2017, but still getting the issue on the latest version.

**Comment by kalessil** [comment]
Fixed!
